### DISCUSSION TITLE
Multiple proposals going against each other

### DISCUSSION THREAD

**Discussion by uuvis** [snapshot-labs/snapshot#1565] [Q&A]
What is the final result in a case when, for example, there are two proposals - first "to keep the project going", and second "to close the project" and the people vote YES for both of them, then in this case people have voted to close and in the same time to keep the project. What's going to happen to the said project then?

**Comment by metrox-eth** [comment]
I guess it depends if the proposal is a temperature checking or if it needs to lead to a decision. Let's say that a proposal is a quick question I assume it's more of a temperature checking. If the proposal is detailed and suggest that it's a decision making proposal its more problematic. In direct democracy we have initiatives and referendums. Interesting topic.

**Reply by uuvis** [reply]
This case which I am referring to is definitely not a temperature checker. We have one proposal from the project founders and few other proposals created by other board members and community as well.
